docs/configuration.asciidoc

```diff
@@ -933,3 +933,33 @@ During startup the Node.js agent queries the local environment to determine whet
 - `none`: Do not query for any cloud provider information.
 
 If the value is not one of the five listed above, the agent will use the value of `auto`.
+
+[[ignore_message_queues]]
```

Comment:
Typically we use hyphens for the HTML anchors.

Aside: Do you have suggestion/ideas for where to include the actual string "ignore_message_queues" in changes like this? The Node.js Agent spells its config vars in the different ignoreMessageQueues pattern which breaks [the grep test](http://jamie-wong.com/2013/07/12/grep-test/). This need not block this change, however.

Reply:
👍 switching to hyphens.  I don't have any good suggestions/ideas for where the literal text `ignore_message_queues` should go.  The decision to stick with camel casing binds our hands there.  I think the best we can do is the more recent convention of prepending `ELASTIC_` to get the env variable name (which presumes a `grep -i` test).